databricks/koalas/namespace.py

```diff
@@ -2599,6 +2600,52 @@ def broadcast(obj) -> DataFrame:
     return DataFrame(obj._internal.with_new_sdf(F.broadcast(obj._internal.spark_frame)))
 
 
+def read_orc(
+    path,
+    columns: Optional[List[str]] = None,
+    index_col: Optional[Union[str, List[str]]] = None,
+    **options
+) -> "DataFrame":
+    """
+    Load an ORC object from the file path, returning a DataFrame.
+
+    Parameters
+    ----------
+    path : str
+        The path string storing the ORC file to be read.
+    columns : list, default None
+        If not None, only these columns will be read from the file.
+    index_col : str or list of str, optional, default: None
+        Index column of table in Spark.
+    options : dict
+        All other options passed directly into Spark's data source.
+
+    Returns
+    -------
+    DataFrame
+
+    Examples
+    --------
+    >>> ks.read_orc('data.orc')  # doctest: +SKIP
```

Comment:
Shall we follow the parquet example?

Reply:
@ueshin In the parquet example, they use `to_parquet` to write parquet file.

But we have no API to write the orc file like `to_orc`, so seems like we cannot mimic the parquet example.

If we want to write the orc, we should use `to_spark()` as below.

```python
>>> ks.range(1).to_spark().write.orc("data.orc")
>>> ks.read_orc("data.orc")
   id
0   0
```

Do you think It's reasonable way to show an example ??

Reply:
Okay. let's implement `to_orc`. Sounds like pandas didn't implement `to_orc` (https://github.com/pandas-dev/pandas/issues/25229#issuecomment-594584996) but Spark supports natively.

Reply:
Okay, I'll follow up. Thanks!